app/Jobs/UpdateQueryserviceAllowList.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Wiki;
+use Maclof\Kubernetes\Client;
+use Maclof\Kubernetes\Models\ConfigMap;
+
+class UpdateQueryserviceAllowList extends Job
+{
+    public function handle(Client $k8s)
+    {
+        $allowList = implode(
```

Comment:
took me a moment to parse this on a friday afternoon but this is very elegant! :sparkling_heart: